pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -144,6 +147,7 @@ let
 
     buildInputs = [
       jemalloc
+      (if mklSupport then mkl else openblas)
```

Comment:
openblas isn't used by tensorflow, so this should only be mkl

Reply:
```suggestion
      (if mklSupport then mkl else null)
```

Reply:
Pushed a change that addresses this as well as the assert and arguments above.  Retesting this now.